.github/workflows/graalvm.yml

```
@@ -46,57 +46,45 @@ jobs:
     runs-on: ${{ needs.start-runner.outputs.label }}
     strategy:
       matrix:
-        java: [ '11' ]
+        java: [ '17' ]
+        graalvm: ['latest', 'dev']
```

remove `dev` since currently does not work with `17`. 

```suggestion
        graalvm: ['latest']
```